_includes/manuals/1.13/5.1.8_oauth.md

```diff
@@ -0,0 +1,35 @@
+Alternatively to basic authentication, limited OAuth authentication is supported
+in the API.
+
+#### Configuration of OAuth in Foreman
+
+OAuth must be enabled in Foreman settings. In *Administer > Settings > Auth*,
+search for *OAuth active* configuration and set it to *Yes*. Then set *OAuth consumer key*
+to some string. This will be a token used by all OAuth clients.
+
+If you want all API requests made using OAuth to be authorized as built-in anonymous
+admin user keep *OAuth map users* set to *No*. If you want to specify the user
+under which the request is made, change this configuration option to *Yes*. This 
+allows client to send FOREMAN-USER header with the login of existing Foreman user.
+Please note that this header is not signed in OAuth request so can be forged.
+Anyone with valid consumer key can impersonate any Foreman user.
+
+#### Request example
+
+Usually some OAuth client library is used to generate the request. An example of
+curl command can be found here to better understand how it works
+
+    $ curl
+      -X GET 'https://foreman.example.tst/api/architectures' \
+      -H 'Content-Type: application/json' \
+      -H 'ForemanApiVersion: 2' \
```

Comment:
works for me for ages as well as -H 'Accept: application/json,version=2;' will change to keep it consisten